tensorflow/python/ops/math_ops.py

```diff
@@ -2786,7 +2786,17 @@ def add_n(inputs, name=None):
   """Adds all input tensors element-wise.
 
   Converts `IndexedSlices` objects into dense tensors prior to adding.
-
+  tf.math.add_n performs same operation as'tf.math.accumulate_n' but it waits
+  for all of its inputs to be ready before beginning to sum, that consumes more 
+  memory.
```

Comment:
I'd suggest adding more detail on why the mem usage is higher. For example:

> This can increase memory if inputs are ready at different times, since minimum temporary storage is proportional to inputs size rather than the output size.